File: JavaScript/Lv.0_더 크게 합치기.js
// JavaScript

function solution(a, b) {
    var answer = 0;
    if (`${a}`+`${b}` >= `${b}`+`${a}`) {
    answer = `${a}`+`${b}`;
    console.log(answer);    
    } else {
        answer = `${b}`+`${a}`;
        console.log(answer);
    }
    
    
    return parseInt(answer);
}


// Math.max 함수를 써도 무방함 EX) return Math.max(`${a}${b}`, `${b}${a}`);

File: Java/Lv.0_더 크게 합치기.java
// Java

class Solution {
    public int solution(int a, int b) {
        int value1 = Integer.parseInt(a+"" + b+"");
        int value2 = Integer.parseInt(b+"" + a+"");
        
        if (value1 >= value2) {
            return value1;
        } else {
            return value2;
        }
    }
}
